### ISSUE TITLE
custom colours for venn

### ISSUE THREAD

**Issue by nweerasu** [ChiLiubio/microeco#51]
Hello again,

I'm wondering if you are able to easily implement a plot_venn colours options to change it from the default Dark2 (similar to what's provided in your other trans_ objects.

I am trying to analyze two different datasets in parallel and it'd be nice to have different colours to aid in differentiation.

Thanks!

**Comment by ChiLiubio** [comment]
Hi,

There is a colour option color_circle in the plot_venn() function of trans_venn. Is it what you need?

Chi

**Comment by nweerasu** [comment]
Sorry, dumb question. Yes that's perfect.
